client/document/browsehappy.jsx

```diff
@@ -24,17 +34,22 @@ function Browsehappy( { entrypoint, dashboardUrl } ) {
 									<EmptyContent
 										illustration="/calypso/images/drake/drake-browser.svg"
 										title="Unsupported Browser"
-										line={ [
-											'Unfortunately this page cannot be used by your browser. You can either ',
-											<a key="dashboard-link" href={ dashboardUrl }>
-												use the classic WordPress dashboard
-											</a>,
-											', or ',
-											<a key="browsehappy-link" href="https://browsehappy.com">
-												upgrade your browser
-											</a>,
-											'.',
-										] }
+										line={
+											<>
+												<p>
+													Unfortunately this page cannot be used by your browser. You can either{ ' ' }
+													<a key="dashboard-link" href={ dashboardUrl }>
+														use the classic WordPress dashboard
+													</a>
+													, or{ ' ' }
+													<a key="browsehappy-link" href="https://browsehappy.com">
+														upgrade your browser
```

Comment:
"upgrade" has the connotation that the suggested browsers on this list are better than the one the user is currently using. Which may well be a modern browser, just one we don't explicitly support.

How about "switch to a supported browser"?